### ISSUE TITLE
Issue Report: Scrollable Sidebar Not Functioning as Expected

### ISSUE THREAD

**Issue by otabekoff** [vuejs/devtools-next#350]
# Description:
Upon testing the scrollable sidebar functionality on the localhost:5173 test page, the expected behavior was not observed. Despite attempting various actions such as checking and unchecking the checkbox option in settings, the sidebar tabs remained unscrollable. Additionally, neither did the tabs transition into the popover menu of the floating panel, nor did the "more" button appear. The sidebar remained unresponsive to changes in the "Scrollable sidebar" option value in the settings menu.

# Steps to Reproduce:

1. Navigate to the localhost:5173 test page.
2. Attempt to interact with the sidebar tabs, including checking and unchecking the checkbox option.
3. Observe that the sidebar tabs do not become scrollable as expected.
4. Click the "more" button to access additional tabs if it appears. However, no additional tabs are displayed, and items in the list do not transition into the popover menu of the floating panel.

# Expected Behavior:

a) The sidebar tabs should become scrollable when the content exceeds the available space.
b) Clicking the "more" button should reveal additional tabs within the popover menu of the floating panel.
c) This process should happen automatically when the checkbox is checked.

The described behavior worked initially but stopped functioning after changing the UI scale to "Huge."

![Expected Behavior](https://github.com/vuejs/devtools-next/assets/60849894/a577ad27-06b0-490e-a95e-c4eeff3d8f93)

# Observed Behavior:

- The sidebar tabs remain unscrollable regardless of the content length.
- Clicking the "more" button does not display additional tabs within the popover menu.

# Attempts to Resolve:

- Checked and unchecked the checkbox option in the settings.
- Manually adjusted the height and width of the VueDevtools 7 panel.

# Additional Information:

- Environment: localhost:5173 test page
- Browser: Microsoft Edge latest stable version
- Vue DevTools: v7.0.27

# Screenshots/Video:
![Observed Behavior](https://github.com/vuejs/devtools-next/assets/60849894/5ff3af6c-62c1-4688-95d7-0d45b7c39aac)

# Priority:
Unknown

# Assigned To:
Unknown

# Deadline:
Unknown

Notes:
Please address this issue in upcoming versions. Feel free to use this issue text style as a template for reporting.

**Comment by alexzhang1030** [comment]
Thank you for reporting, we will investigate it soon

**Comment by alexzhang1030** [comment]
Hey, I'm using `vite-plugin-vue-devtools@7.1.3`, unfortunately, I cannot reproduce it. 

Can you upgrade the version and check it out agaign or provide more info here?(maybe a video)

https://github.com/vuejs/devtools-next/assets/49969959/6eb07aa4-e3bd-4fd9-9d08-fa720383156a

**Comment by otabekoff** [comment]
I am making my YouTube video. I've also mentioned about this in that video. I'll record a video later and leave here. Thanks.

**Comment by alexzhang1030** [comment]
Any progress? Is this issue still ongoing?

**Comment by otabekoff** [comment]
Yes, but I couldn't figure out what's happening. Recently, I had to reset my Windows 11 for a valid reason. I wanted to try a fresh OS and browser. It worked initially, but when I returned later, the issue persisted like before.

**Comment by alexzhang1030** [comment]
When you have time, please upload a video here. This will help us quickly identify the issue.
